scarb/tests/build_cairo_plugin.rs

```diff
@@ -84,6 +84,9 @@ fn simple_project_with_code(t: &impl PathChild, code: impl ToString) {
         [dependencies]
         cairo-lang-macro = {{ path = {macro_lib_path}}}
         cairo-lang-macro-stable = {{ path = {macro_stable_lib_path}}}
+        serde = {{ version = "*", features = ["derive"] }}
+        serde_json = "*"
+        linkme = "0.3"
```

Comment:
why do you need to link `linkme` here?

Reply:
Because the macro from `cairo-lang-macro-attributes` puts the `linkme::distributed_slice` call into the users project 😨

Reply:
let's `#[doc(hidden)] pub use` this item then